pylinalg/func/matrix.py

```diff
@@ -493,8 +494,41 @@ def matrix_make_look_at(eye, target, up, /, *, out=None, dtype=None):
     Returns
     -------
     rotation_matrix : ndarray, [4, 4]
-        A matrix describing the rotation.
+        A homogeneous matrix describing the rotation.
+
+    Notes
+    -----
+    The implementation ignores the scale of ``target - eye`` and ``up``, meaning
+    that the resulting rotation makes the coordinate frame axes _point_ in the
+    direction but won't scale the units to _match_ the direction.
```

Comment:
> but won't scale the units to _match_ the direction.

What does this mean? If a user provides vectors that are not normalized, will it be the same as if they were normalized or not? Let's update this note so that this is clear.

Reply:
It's more complicated then that, unfortunately. In the current implementation, the scale of `up` is ignored (though we want to change how `up` works anyway, so this is subject to change). We need to normalize `target-eye`, so its scale doesn't matter. The (relative) scale of `target` and `eye`, however, controls the direction of the new z-axis so these scales very much matter.

Hence the result will be different if all input vectors are normalized vs non-normalized. Maybe we just delete the note as it is more confusing than insightful?

Reply:
Now I'm confused about what `target` and `eye` mean. I initially thought they were positions that would together define the new view direction.

Reply:
Yes, thats what they do. However this is implemented as 

```python
new_z_axis = target - eye
new_z_axis /= np.linalg.norm(new_z_axis)
```

So the resulting vector is normalized and doesn't change the scale of an object. However, the actual direction is different if we use `target - eye` or `target - 2*eye` but the same if we use `target - eye` or `2*target - 2*eye`. Thus the (relative) scaling of `target` and `eye` matters.